### DISCUSSION TITLE
How to binding to textblock's inlines?

### DISCUSSION THREAD

**Discussion by mcguu2020** [microsoft/WindowsAppSDK#2932] [Q&A]
```
<TextBlock IsTextSelectionEnabled="True" SelectionHighlightColor="Green" FontFamily="Arial">
    <Run Foreground="Blue" FontWeight="Light" Text="This text demonstrates "></Run>
    <Span FontWeight="SemiBold">
        <Run FontStyle="Italic">the use of inlines </Run>
        <Run Foreground="Red">with formatting.</Run>
    </Span>
</TextBlock>
```

I want to make this in a listview and the inlines is binding to a dynamic inline.

`<TextBlock IsTextSelectionEnabled="True" SelectionHightlightColor="Green" FontFamily="Arial" Inlines="{Binding Inline}"/>
`
`Inline` is a `InlineCollection`

```
public InlineCollection Inline
{
     get
    {
          var run =new Run(){Text="This text demonstrates"};
          var run1=new Run(){Text="the use of inlines"};
          var inlines=new InlineCollection();
         inlines.Append(run);
         inlines.Append(run1);
        return inlines.
    }
}
```
but the `InlineCollection `is not bindable.

**Comment by KiyanYang** [comment]
Reference [TextBlock.Inlines Property (Microsoft.UI.Xaml.Controls) - Windows App SDK | Microsoft Learn](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.textblock.inlines?view=windows-app-sdk-1.1), Inlines Property only can get.

```cs
public InlineCollection Inlines { get; }
```
